### ISSUE TITLE
Adding Subify in the Windows Context menu

### ISSUE THREAD

**Issue by AndreasFurster** [matcornic/subify#8]
I just managed to get Subify in the windows context menu. Although it would be nice if this can be done by the application itself. 

How I did it:

1. Open Run (win + R) and execute `shell:sendto` (this will navigate to the folder with the "Send to" context menu shortcuts)
1. Add an new shortcut
1. Location: `C:\Windows\System32\cmd.exe /k "C:\tools\subify.exe dl -o"` (Change the subify command if needed)
1. Choose an name, I took "Subtitle and open"

And you're done! Just right mouse click on an video and `Send to > Subtitle and open`. It will find the subtitle and open the video.

Issues:

- You have to add the first part to prevent the `You need to open cmd.exe and run it from there.` warning. This makes sense if there is no file parameter, but the `Send to` functionality in explorer automatically adds this.
- You can not change the icon. (Changing it will replace the CMD shortcut icon everywhere)
- After opening there is still an CMD window open. You can fix this by changing `/k` to `/c`, but then you will never know when an error occurs.

My request:
Please implement native context menu functionality or adjust the `You need to open cmd.exe and run it from there.` warning. This will also solve all issues I have.

**Comment by matcornic** [comment]
Yes, this is something interesting that could be done for Mac as well.

In the meantime, can you update the README to add the same kind of section than the one for Mac (which uses Services Automator) ?

**Comment by matcornic** [comment]
FYI, I added your tips in the README.
